hack/gosec.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+source hack/common.sh
+export ARTIFACTS=$KUBEVIRT_DIR/${ARTIFACTS:-_out/artifacts}
+if ${GENERATE:-"false"} == "true"; then
```

Comment:
These two steps are called directly after each other in the Makefile. Would it make sense to remove the conditional and just execute it one after the other directly?

Reply:
But IIRC `git diff` does not return a non zero exit code? So why would the diff fail?

Reply:
ah, you are right...